src/pages/publication/description_tab/projects_field/helpers.ts

```diff
@@ -0,0 +1,56 @@
+import { CristinProject, ResearchProject } from '../../../../types/project.types';
+import { LanguageCodes } from '../../../../types/language.types';
+import { BackendTypeNames } from '../../../../types/publication_types/commonPublication.types';
+
+export const getProjectTitle = (option: CristinProject) => {
+  const selectedLanguage = localStorage.getItem('i18nextLng');
+  if (selectedLanguage === LanguageCodes.NORWEGIAN_BOKMAL || selectedLanguage === LanguageCodes.NORWEGIAN_NYNORSK) {
+    const norwegianTitle = option.titles.find((title) => title.language === 'no')?.title;
+    if (norwegianTitle) {
+      return norwegianTitle;
+    }
+  }
+  if (selectedLanguage === LanguageCodes.ENGLISH) {
+    const englishTitle = option.titles.find((title) => title.language === 'en')?.title;
+    if (englishTitle) {
+      return englishTitle;
+    }
+  }
+  return option.titles[0].title;
+};
+
+export const getProjectTitleParts = (project: CristinProject, searchTerm: string) => {
+  const title = getProjectTitle(project);
+  const indexOfMatch = title.toLocaleLowerCase().indexOf(searchTerm);
+
+  const parts =
+    indexOfMatch === -1
+      ? [title]
+      : [
+          title.substr(0, indexOfMatch),
+          title.substr(indexOfMatch, searchTerm.length),
+          title.substr(indexOfMatch + searchTerm.length),
+        ];
+  return parts;
+};
+
```

Comment:
vi må ha noe mapping som under her frem til vi har en felles modell for prosjekt: https://unit.atlassian.net/browse/NP-1718